packages/react-router/docs/api/withRouter.md

```diff
@@ -27,3 +27,14 @@ class ShowTheLocation extends React.Component {
 // terminology) to the router.
 const ShowTheLocationWithRouter = withRouter(ShowTheLocation)
 ```
+
+**IMPORTANT NOTE:** If you are using withRouter to prevent shouldComponentUpdate blocks, it is important that withRouter wraps the component that implements shouldComponentUpate. For example, when using Redux: 
```

Comment:
I would place backticks around the code (`shouldComponentUpdate` and `withRouter`)